docs/blog/_posts/2019-12-20-21th-dotty-milestone-release.md

```diff
@@ -0,0 +1,378 @@
+* `scala.quoted.matching.ExprSeq`: matches an explicit sequence of expresions and returns them. These sequences are useful to get individual `Expr[T]` out of a varargs expression of type `Expr[Seq[T]]`.
+* `scala.quoted.matching.ConstSeq`:  matches an explicit sequence of literal values and returns them.
+
+The following snippet demonstrates the new _quoted patterns_ implementing a
+simple, 1-level, non-recursive rewriter macro for exponents. `rewrite` is a an
+inline method definition designating a macro as usual. To inspect an `expr`
+value with friendly syntax we can now use the quoted syntax as patterns inside a
+match expression. Notice that quotes designate patterns and `$`, the familiar
+syntax for splices is used to _extract_ (capture) information out of a pattern.
+
+```scala
+inline def rewrite(expr: => Double): Double = ${rewrite('expr)}
+
+def rewrite(expr: Expr[Double])(given QuoteContext): Expr[Double] = {
+  val res = expr match {
+    // product rule
+    case '{ power2($a, $x) * power2($b, $y)} if a.matches(b) => '{ power2($a, $x + $y) }
+    // rules of 1
+    case '{ power2($a, 1)} => a
+    case '{ power2(1, $a)} => '{ 1.0 }
+    // rule of 0
+    case '{ power2($a, 0)} => '{ 1.0 }
+    // power rule
+    case '{ power2(power2($a, $x), $y)} => '{ power2($a, $x * $y ) }
+    case _ => expr
+  }
+  res
+}
+```
+
+To learn more read our docs on [pattern matching over quotes](https://dotty.epfl.ch/docs/reference/metaprogramming/macros.html#pattern-matching-on-quoted-expressions).
+
+## Added support for SemanticDB extraction
+
+Scala 3 now offers support for SemanticDB extraction producing `.semanticdb`
+files. Users can use the `-Ysemanticdb` compiler flag to enable exporting
+SemanticDB files. Providing `-semanticdb-target` allows the user to select a
+separate target destination for the `META-INF` directory that stores SemanticDB
+files and `-sourceroot` to calculate a relative path for SemanticDB files within
+`META-INF`.
+
+# Let us know what you think!
+
+If you have questions or any sort of feedback, feel free to send us a message on our
+[Gitter channel](https://gitter.im/lampepfl/dotty). If you encounter a bug, please
+[open an issue on GitHub](https://github.com/lampepfl/dotty/issues/new).
+
+## Contributing
+
+Thank you to all the contributors who made this release possible!
+
+According to `git shortlog -sn --no-merges 0.20.0-RC1..0.21.0-RC1` these are:
+
+```
+   176  Martin Odersky
+   171  Bryan Abate
+    88  Nicolas Stucki
+    81  Jamie Thompson
+    44  noti0na1
+    29  Anatolii
+    28  bishabosha
+    23  Antoine Brunner
+    19  Guillaume Martres
+    10  Aleksander Boruch-Gruszecki
+     8  Guillaume Raffin
+     6  Olafur Pall Geirsson
+     5  Sébastien Doeraene
+     4  Aggelos Biboudis
+     4  Liu Fengyun
+     4  Paolo G. Giarrusso
+     3  Andrea Mocci
+     3  Martijn Hoekstra
+     2  Ben Elliott
+     2  Patrik Mada
+     2  Rafal Piotrowski
+     2  odersky
```

Comment:
which one :-)

(a `.mailmap` addition would sort this out, same also for the multiple Anatoliis)

Reply:
@anatoliykmetyuk can you take care of that?